binstar_client/commands/download.py

```diff
@@ -0,0 +1,64 @@
+'''
+Usage:
+    anaconda notebook download notebook
+    anaconda notebook download user/notebook
+'''
+
+from __future__ import unicode_literals
+
+import argparse
+import logging
+
+from binstar_client import errors
+from binstar_client.utils import get_binstar
+from binstar_client.utils.notebook import Downloader, parse, has_environment
+
+log = logging.getLogger("binstar.download")
+
+
+def add_parser(subparsers):
+    description = 'Download notebooks from Anaconda Cloud'
```

Comment:
seriously? this is gonna be the first `add_parser` function with a docstring in the history of continuum. C'mon! we know what `add_parser` is doing!